custom-connectors/Tikit/Readme.md

```diff
@@ -0,0 +1,36 @@
+# Tikit [Sample] Connector
+
+Tikit provides a basic ticket management system. Using this API, you can create tickets and files associated with it.Very often, you may want to leverage the business with tickets.This is the solution for it.
+
```

Comment:
@1MrSharma Publisher is missing, Please refer the templates - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Hello,
The publisher is already present in apiProperties.json although had added stack owner property to it.

Reply:
@1MrSharma I am aware that you have added the publisher and stackOwner, It is a documentation file for the users and as per the template, you need to provide the publisher section.

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Updated the changes required.